packages/react/src/components/ToggleGroup/ToggleGroup.tsx

```diff
@@ -34,14 +34,14 @@ export type ToggleGroupProps = {
 /**
  * Display a group of buttons that can be toggled between.
  * @example
- * <ToggleGroup.Root onChange={(value) => console.log(value)}>
+ * <ToggleGroup onChange={(value) => console.log(value)}>
  *   <ToggleGroup.Item value='1'>Toggle 1</ToggleGroup.Item>
  *   <ToggleGroup.Item value='2'>Toggle 2</ToggleGroup.Item>
  *   <ToggleGroup.Item value='3'>Toggle 3</ToggleGroup.Item>
- * </ToggleGroup.Root>
+ * </ToggleGroup>
  */
-export const ToggleGroupRoot = forwardRef<HTMLDivElement, ToggleGroupProps>(
-  (
+export const ToggleGroup = forwardRef<HTMLDivElement, ToggleGroupProps>(
```

Comment:
This has a context and is required for `ToggleGroup` to work, do we want to go without the `Context` suffix because we render a div 🤔

Reply:
I suggest yes; `.Context` is only used when not rendering a element, ref;
https://github.com/digdir/designsystemet/issues/2221#issuecomment-2339897367

For the end user, it is not important wether or not a context is used under the hood, but it is important wether or not a DOM element is rendered :)

Reply:
As long as we have consistent rules. We should probably make this support `asChild` as well

Reply:
That however means we have to move the hidden input outside of the wrapping div 🤔

Reply:
Why do we need to support `asChild` on the wrapper? ☺️

Reply:
It's just for consistency, but we can look at this later on.